Apps/WebClient/src/Server/Models/UserPreferenceModel.cs

```diff
@@ -16,17 +16,14 @@
 namespace HealthGateway.WebClient.Models
 {
     using System;
+    using System.Collections.Generic;
+    using System.Linq;
 
     /// <summary>
     /// Model that provides a user representation of an user preference database model.
     /// </summary>
     public class UserPreferenceModel
```

Comment:
Why are we creating a front end model?  I'm not seeing a lot of value.

Reply:
@sslaws do you mean we can just expose the dbmodel which contains the Preference and Value pair in the Service & Controller right?